.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+# Checklist
+
+- Branch
+    - [ ] Commit sequence broadly makes sense
+    - [ ] Commits have useful messages
+    - [ ] New tests are added if needed and existing tests are updated
+    - [ ] If this branch changes Consensus and has any consequences for downstream repositories or end users, said changes must be documented in [`interface-CHANGELOG.md`](../ouroboros-consensus/docs/interface-CHANGELOG.md)
+    - [ ] If this branch changes Network and has any consequences for downstream repositories or end users, said changes must be documented in [`interface-CHANGELOG.md`](../docs/interface-CHANGELOG.md)    
+- Pull Request
```

Comment:
I think "Self-reviewed the diff" and "Reviewer requested" seem a bit excessive, but that's not to say they won't be helpful occasionally.

Reply:
Yeah, maybe, but it is not that bad to check two boxes and might be helpful sometimes